### PR TITLE
Add and organise proposals from last year

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* adding and organising funding or other kinds of proposals

### List of changes proposed in this PR (pull-request)

* Add markdown files

### What should a reviewer concentrate their feedback on?

- [ ] Is anything missing?
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: project_management/proposals/2022-04-Einstein-Award-Submission.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
...r success lies in the diversity of contributors, the contributions they make to the project and the impact seen on the wider STEM sector. Managed by a core team of 18 members from The Alan Turing Institute and more widely from the international volunteer community, _The Turing Way_ now boasts over 325 contributors, who have collectively written over 260 pages across 54 chapters to date. All collaborations and development processes occur in the open, challenging the disconnected and isolated nature of the traditional academic research process. Contributors include researchers, open science practitioners, educators, policymakers and data experts across academia, government and industry, in various domains, at all levels of seniority, representing countries including Australia, Argentina, Brazil, Germany, India, the Netherlands, Saudi Arabia, South Africa, Uganda, UK, USA, Venezuela and beyond. Based on web traffic monitoring coupled with references in more than 3500 monthly visitors, 100+ online publications, 30+ citations in peer-reviewed articles and personal testimonials, The Turing Way has thousands of users worldwide in academia, industry, open communities and the public sector.
```
Comment (aleesteele) [inline project_management/proposals/2022-04-Einstein-Award-Submission.md:15]:
```suggestion
_The Turing Way_ project is a working example of team science, using a wide community to define the standards and expectations the community would like to see in responsible, ethical and quality research practice. _The Turing Way_ guides seek to provide all the information that researchers need at the start of their projects to ensure that they are conducting research that is easy to reuse and reproduce at the end. _The Turing Way_ team hasn’t produced these guides by choosing a select group of people to decide what is “best practice”. Their success lies in the diversity of contributors, the contributions they make to the project and the impact seen on the wider STEM sector. Managed by a core team of 18 members from The Alan Turing Institute and more widely from the international volunteer community, _The Turing Way_ now boasts over 325 contributors, who have collectively written over 260 pages across 54 chapters to date. All collaborations and development processes occur in the open, challenging the disconnected and isolated nature of the traditional academic research process. Contributors include researchers, open science practitioners, educators, policymakers and data experts across academia, government and industry, in various domains, at all levels of seniority, representing countries including Australia, Argentina, Brazil, Germany, India, the Netherlands, Saudi Arabia, South Africa, Uganda, UK, USA, Venezuela and beyond. Based on web traffic monitoring coupled with references in more than 3500 monthly visitors, 100+ online publications, 30+ citations in peer-reviewed articles and personal testimonials, The Turing Way has thousands of users worldwide in academia, industry, open communities and the public sector. 
```

File: project_management/proposals/2021-hiddenref.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# HiddenRef submission for The Turing Way - successsful!
+
+- Category: Practices
+- Submitter: Emma Karoune
+
+The Turing Way’s goal is to provide all the information that researchers need at the start of their projects to ensure that they are easy to reproduce at the end. It is an open-source project that involves and supports its diverse community to make data science reproducible, ethical, collaborative and inclusive for everyone.
```
Comment (aleesteele) [inline project_management/proposals/2021-hiddenref.md:6]:
```suggestion
_The Turing Way_’s goal is to provide all the information that researchers need at the start of their projects to ensure that they are easy to reproduce at the end. It is an open-source project that involves and supports its diverse community to make data science reproducible, ethical, collaborative and inclusive for everyone. 
```

File: project_management/proposals/2021-hiddenref.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+The project was conceived and is led by Kirstie Whitaker, the lead of the Tools, Practices and Systems Programme at The Alan Turing Institute. The Community Manager, Malvika Sharan, plays an important role in bringing together the community at regular events and nurturing contributions to the ever-expanding resource, an online book. 
+The book currently has five guides (Reproducibility, Project Design, Communication, Collaboration and Ethical Research), and a Community handbook. Each guide is a work in progress, as we encourage everyone to keep adding to this resource.
+However, The Turing Way is much more than a book: it is a community of practice based around the moonshot goal of making reproducible research ‘too easy not to do’. The outputs belong to The Turing Way community and everyone can freely read, reuse, distribute, modify, and contribute back to the resources in the book. The project is built on open source infrastructure such as Git, Jupiter Book and Netlify.
```
Comment (aleesteele) [inline project_management/proposals/2021-hiddenref.md:10]:
```suggestion
However, _The Turing Way_ is much more than a book: it is a community of practice based around the moonshot goal of making reproducible research ‘too easy not to do’. The outputs belong to The Turing Way community and everyone can freely read, reuse, distribute, modify, and contribute back to the resources in the book. The project is built on open source infrastructure such as Git, Jupiter Book and Netlify.
```

**Comment by bsipocz** [comment]
We are in the middle of reorganizing the repository and as part of that work have moved some of the documents into a new repo. This has introcude code conflicts above, and therefore I am closing this pull request.

Could you please reopen this PR in the new PM repo: https://github.com/the-turing-way/project-management

**Comment by JimMadge** [comment]
This PR has been moved to [here](https://github.com/the-turing-way/project-management/pull/1) :pray:.

**Pull request closed without merging**